src/Style/Sass/pages/_auth.scss

```diff
@@ -0,0 +1,59 @@
+.common-form {
+  margin: 0 auto;
+  border-radius: 0.3em;
+  box-shadow: 0 4px 4px $form-box-shadow-color;
+  color: $text-color-default;
+  height: fit-content;
+  margin: 0 auto;
+  margin-top: 50px;
+  padding: 15px;
+}
+
+.common-form-button {
+  background: $form-button-color;
+  border-radius: 40px;
+  color: $iconcolor;
+  margin: 0 auto;
+  font-size: 25px;
+  font-family: "lato"
+}
+
+.common-form-input {
+  height: 50px;
+  font-size: 18px;
+  background: #ebebeb;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  margin-bottom: 35px;
+}
+
+.m-3{
```

Comment:
this is a bootstrap class and it is used for margin, this means anyone who will use it will be getting this `font-size`